### ISSUE TITLE
Feature Request: More levels of difficulty

### ISSUE THREAD

**Issue by pychess** [pychess/pychess#545]
Original [issue 545](https://code.google.com/p/pychess/issues/detail?id=545) reported by [zollo.john](https://code.google.com/u/117741568078251192015/) 2010-03-23

Hello everyone!!!  I am very impressed with r1562!  It is stable and works
very well!!!  

I had a feature request though.  I play Chessmaster sometimes, and I'm
impressed by its MANY MANY levels of difficultly.  There are dozens of
difficultly settings with personalities for each one.  Although I don't
personally need personality profiles (like aggressive vs. drawish and
positional vs. tactical) I do like having many difficulty settings.

I was wondering if it was possible to have around 20 difficulty settings in
PyChess instead of the current 10.  I know this seems ridiculous, but I
really like the idea of incrementally clicking up the difficulty.  For
instance, in 5 minute blitz, I can win repeatedly on level 3, but I have a
tough time with level 4.  For me, I would sincerely appreciate a level 3.5!!!

If there's anything you could do about it I would sincerely appreciate it.
I know eveyone is probably very busy, so anything you could do would
sincerely help!!!

Thanks!!!

take care!

Sincerely,
John

**Comment by pychess** [comment]
Original [comment 1.](https://code.google.com/p/pychess/issues/detail?id=545#c1) posted by [gbtami](https://code.google.com/u/gbtami/) 2010-04-20
- added label: Enhancement

**Comment by pychess** [comment]
Original [comment 2.](https://code.google.com/p/pychess/issues/detail?id=545#c2) posted by [leogregianin](https://code.google.com/u/leogregianin@gmail.com/) 2011-01-28

The ICC Dasher has 14 difficulty levels based on rating of 1000, 1100, 1200, 1400, 1600, 1800, ... 2700.

**Comment by pychess** [comment]
Original [comment 3.](https://code.google.com/p/pychess/issues/detail?id=545#c3) posted by [lobais](https://code.google.com/u/lobais/) 2011-01-28

This is an interesting problem.
On one hand you make a very valid point that we want players to get the exact right level of challenge.
On the other hand we can't just add ten more levels to the slider. It is not wide enough, and even if it was wider, it would make it less easy to figure out what level to play at initially.
Ideally we need a way to track the strength of the player, so to give him or her more fine grained control around the level that is relevant.

I hope fics can give you the right level of challenge in the mean time.

**Comment by pychess** [comment]
Original [comment 4.](https://code.google.com/p/pychess/issues/detail?id=545#c4) posted by [gbtami](https://code.google.com/u/gbtami/) 2013-03-23

Another request from our list:
On Saturday, March 23, 2013 11:26:05 AM UTC+1, Adam Sveshnikov wrote:

```
Maybe only 1 suggestion, to have possibility to adjust engine.s for scaled rated (1000-2800) play, not this 1-8 levels situation. (something that shredder classic for linux have) [ but is not must, is fine like this ]

Aleksandar
Linux Mint Cinnamon
http://sourceforge.net/users/josifoski
```
- changed status to: Accepted

**Comment by pychess** [comment]
Original [comment 5.](https://code.google.com/p/pychess/issues/detail?id=545#c5) posted by [gbtami](https://code.google.com/u/gbtami/) 2013-05-07

This issue was closed by revision 577551f431c6.
- changed status to: Fixed
